jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/loginform/JmixLoginI18n.java

```diff
@@ -30,14 +30,16 @@ public class JmixLoginI18n extends LoginI18n {
 
     protected static final JsonValue JMIX_DEFAULT_I18N;
 
+    private JmixForm jmixForm;
+
     /*
      * CAUTION! Copied from com.vaadin.flow.component.login.LoginI18n
      */
     static {
         try {
             final JsonFactory JSON_FACTORY = new JreJsonFactory();
             JMIX_DEFAULT_I18N = JSON_FACTORY.parse(
-                    IOUtils.toString(LoginI18n.class.getResource("i18n.json"),
+                    IOUtils.toString(JmixLoginI18n.class.getResource("i18n.json"),
```

Comment:
Since we have fully overrided this object, why we should use JSON file for storing defaults? Would not be simpler to define them in `createDefault()` while building an object?